src/common/string_util.cpp

```diff
@@ -199,3 +199,6 @@ string StringUtil::Strip(const string &str, char c) {
 	tmp.erase(std::remove(tmp.begin(), tmp.end(), c), tmp.end());
 	return tmp;
 }
+string StringUtil::Concat(const string &str, int64_t value) {
```

Comment:
Is there a reason we need this concat function?

Reply:
I was using it before. I'm going to remove it.